src/util/redisConnector.js

```diff
@@ -59,20 +64,32 @@ const RedisDB = {
    * @returns value which can be json or string or number
    *
    */
-  async getVal(key, isJsonExpected = true) {
+  async getVal(key, isObjExpected = true) {
```

Comment:
Please update the JS docs its still showing `description` for `isJsonExpected`

Reply:
Okay

Reply:
done